app.rb

```diff
@@ -11,6 +11,36 @@ def initialize
     @rentals = []
   end
 
+  def display_menu
+    puts "\nPlease choose an option by entering a number:"
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Add a person'
+    puts '4 - Add a book'
+    puts '5 - Add a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def choose_option(option)
+    case option
+    when 1
+      list_all_books
+    when 2
+      list_all_people
+    when 3
+      create_person
+    when 4
+      add_book
+    when 5
+      add_rental
+    when 6
+      list_all_rentals
+    else
+      'Wrong number please enter a number between 1 and 7'
+    end
+  end
+
```

Comment:
- You guys have done well so far, but please note that you have not properly refactored all of your code, as you'd notice the app.rb file is still bulky and does not follow the [single-responsibility-principle](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid), which helps us ensure that our class or module should be responsible for a single action. According to the project requirements, I suggest that you extract some of your logic in the app.rb into separate classes and ensure that each of your new classes follows the SRP. Please fix this 🙏